magento/magento-cloud-docker
# MCLOUD-5752: Healthchecks for nginx and php fpm

### Description

[MCLOUD-5752](https://jira.corp.magento.com/browse/MCLOUD-5752): Add container health checks for Nginx and PHP-FPM.

### Fixed Issues (if relevant)

1. magento/magento-cloud-docker#116: Add additional health checks.

### Manual testing scenarios

1. Build Nginx and PHP-FPM containers.
2. Run ./vendor/bin/ece-docker build:compose --mode="developer".
3. Run docker-compose up
4. Check if the Nginx and FPM containers are healthy.

### Contribution checklist
